src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_performing_slr_with_min_policy.cs

```diff
@@ -46,7 +45,7 @@ public RetryEndpoint()
                     .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
```

Comment:
Not part of your changes, but with a custom retry policy, is there even a point to specifying a SecondLevelRetriesConfig here? Looks like several tests do this.